### DISCUSSION TITLE
Modify slide notes

### DISCUSSION THREAD

**Discussion by jcoliz** [ShapeCrawler/ShapeCrawler#682] [Ideas]
Is it possible to add/edit slide notes? I could not find this, if exists.

**Comment by AdamShakhabov** [comment]
No, there are not these features available yet.

- [x] https://github.com/ShapeCrawler/ShapeCrawler/issues/684
- [x] https://github.com/ShapeCrawler/ShapeCrawler/issues/685

**Comment by jcoliz** [comment]
Ok, thanks for confirming!
